### PR TITLE
feat(weekly.ci) add role definition and assignment for new disk access

### PR DESCRIPTION
**Pull request by smerle33**
following #700 and as per https://github.com/jenkins-infra/helpdesk/issues/4044
this PR will allow a pod to access the new PV/PVC/DISK and handle the data migration.

this PR was tested locally, it failed when using both ID and name for the role assignment/role definition, it failed with only the id with `Error: authorization.RoleAssignmentsClient#Create: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidRoleDefinitionId" Message="The role definition ID '5a60a9a4-88e7-d46c-6311-6aa2bba1581c|' is not valid."`

but succeeded with only the `role_definition_name` ...

### CODE REVIEW

**Comment by smerle33** [comment]
`Apply complete! Resources: 2 added, 0 changed, 0 destroyed.`
and then
`Destroy complete! Resources: 2 destroyed.`

**Review by dduportal — DISMISSED** [review]
We might need other permissions than `disk/read`: LGTM but worth cehcking if there is a doc. on azure

**Review by dduportal — APPROVED** [review]
`Plan: 2 to add, 0 to change, 0 to destroy.`

LGTM!

**Pull request merged**
